requirements/common.in

```diff
@@ -169,7 +169,7 @@ pyoembed==0.1.2
 # these tightly, including fetching content not included in the normal
 # release tarballs (which is a bug).  So we need to pin it makes sense
 # to pin a version from Git rather than a release.
--e "git+https://github.com/zulip/python-zulip-api.git@0.4.6#egg=zulip==0.4.6_git&subdirectory=zulip"
+-e "git+https://github.com/zulip/python-zulip-api.git@292a3bc0671dd28cde79dd532a86a3a049bfb2f5#egg=zulip&subdirectory=zulip"
```

Comment:
Can you split the API version upgrade to its own commit?

I think @eeshangarg was having some trouble upgrading this due to some weird mypy error, so I'd like to be able to debug and fix that without reading the rest of this.